plugin/plugin_test.go

```diff
@@ -178,7 +194,7 @@ func TestExecInvalidCustomDetails(t *testing.T) {
 	mockClient := new(MockPagerDutyClient)
 	ctx := context.Background()
 	args := Args{
-		RoutingKey:        "testRoutingKey",
+		IntegrationKey:    "testRoutingKey",
```

Comment:
```suggestion
		IntegrationKey:    "testIntegrationKey",
```

Reply:
@Ompragash, renamed the above as per your suggestion. Please check and confirm the changes.